database/photomodel.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+
+mongoose.Promise = global.Promise;
```

Comment:
What's the purpose of this line. I don't see mongoose. Promise used anywhere in this file. Just curious :-)